### DISCUSSION TITLE
Why am I getting an adguard.com cert for a site when I don't have https filtering?

### DISCUSSION THREAD

**Discussion by jerrac** [AdguardTeam/AdGuardHome#7339] [Q&A]
Ok, so I've run into an odd issue that has me confused. 

When I try to visit a certain site, (`app.getresponse.com`), instead of just not working like I'd expect if it was being filtered, I hit a bad https cert error. Specifically that the cert is for `dns.adguard.com, dns-family.adguard.com, dns-unfiltered.adguard.com` and not for the actual domain. 

I get the cert issue in both Firefox and Chromium.

I don't have dns over https configured on any client, and I don't see any adguard.com certs on any other sites I visit over https. The only place I use dns over https is with the default upstream server setting in AdGuard Home: `https://dns10.quad9.net/dns-query`.

I don't, and never have had, any other AdGuard app installed on my pc (or my family members laptop where I first hit this issue). 

I've only ever had AdGuard Home installed on a home server. 

And this particular instance has never had the dhcp server turned on, nor is it the gateway for my traffic. All it is configured to do is serve dns.

So I'm not sure how, or why, I'm hitting an AdGuard https cert. 

I've gone through all the settings pages twice now, and I don't see anything related to https filtering. If my quick check of the docs my web search turned up is any indication, https filtering is not associated with AdGuard Home. Just other AdGuard products. Is that accurate?

The cert I'm getting shown at that site has sha-256 fingerprint of: `A9:7C:52:34:AE:0E:24:20:34:56:DA:FA:9A:BB:B3:51:00:F1:A4:12:6A:12:DE:CE:17:CC:ED:B5:9C:62:C7:E1` if that helps. 

My computer is Ubuntu 24.04, my family members laptop is Windows 10. AdGuard Home is version: v0.108.0-b.58, I just updated it this evening. I have it running alongside OPNSense on a mini pc. Technically most/all of my dns traffic is forwarded from OPNSense's Unbound instance to AdGuard Home.

Anyway, anyone have any clues? Something obvious I'm missing?

(For the record, I'm just very confused over this, it's not actually causing any real harm. I don't actually need to visit that site. I just want to figure out why my stuff isn't working the way I expect. :) )

**Comment by timkgh** [comment]
Maybe you have these options enabled?
![9965](https://github.com/user-attachments/assets/103a44fc-4f09-436d-b447-f575f36448c1)

**Reply by jerrac** [reply]
Both are unchecked.
